es-core/src/components/ImageGridComponent.h

```diff
@@ -305,7 +305,9 @@ void ImageGridComponent<T>::applyTheme(const std::shared_ptr<ThemeData>& theme,
 			std::string path = elem->get<std::string>("gameImage");
 
 			if (!ResourceManager::getInstance()->fileExists(path))
+			{
```

Comment:
Added braces around LOG to remove dangling else warning.

/home/devel/temp/EmulationStation_Stable/es-core/src/components/ImageGridComponent.h:309:4: warning: 
add explicit braces to avoid dangling else [-Wdangling-else]
                        else
                        ^